x-pack/plugins/case/server/services/alerts/index.ts

```diff
@@ -5,24 +5,26 @@
  * 2.0.
  */
 
+import _ from 'lodash';
```

Comment:
nit: Better to use `import { isEmpty } from lodash/fp`

Reply:
👍